log/handler.go

```diff
@@ -81,6 +85,34 @@ func NetHandler(network, addr string, fmtr Format) (Handler, error) {
 	return closingHandler{conn, StreamHandler(conn, fmtr)}, nil
 }
 
+// SplunkHandler posts data to Splunk HTTP Event Collector.
+func SplunkHandler(url, token, source, sourcetype, index string, skipTLSVerify bool, fmtr Format,
```

Comment:
Would be better to move this to the log/splunk package (maybe call it simply `Handler`).
Having the handler constructor in package log means that every other package needs to
link in the splunk client. If you move it, only internal/debug will need the splunk client.

Reply:
Done.